scripts/minikube.sh

```diff
@@ -278,11 +278,11 @@ cephcsi)
     ;;
 k8s-sidecar)
     echo "copying the kubernetes sidecar images"
-    copy_image_to_cluster "${K8S_IMAGE_REPO}"/csi-attacher:v3.0.2 "${K8S_IMAGE_REPO}"/csi-attacher:v3.0.2
-    copy_image_to_cluster "${K8S_IMAGE_REPO}"/csi-snapshotter:v3.0.2 $"${K8S_IMAGE_REPO}"/csi-snapshotter:v3.0.2
-    copy_image_to_cluster "${K8S_IMAGE_REPO}"/csi-provisioner:v2.0.4 "${K8S_IMAGE_REPO}"/csi-provisioner:v2.0.4
-    copy_image_to_cluster "${K8S_IMAGE_REPO}"/csi-node-driver-registrar:v2.0.1 "${K8S_IMAGE_REPO}"/csi-node-driver-registrar:v2.0.1
-    copy_image_to_cluster "${K8S_IMAGE_REPO}"/csi-resizer:v1.0.1 "${K8S_IMAGE_REPO}"/csi-resizer:v1.0.1
+    copy_image_to_cluster "${K8S_IMAGE_REPO}"/csi-attacher:v3.2.1 "${K8S_IMAGE_REPO}"/csi-attacher:v3.2.1
+    copy_image_to_cluster "${K8S_IMAGE_REPO}"/csi-snapshotter:v4.1.1 $"${K8S_IMAGE_REPO}"/csi-snapshotter:v4.1.1
+    copy_image_to_cluster "${K8S_IMAGE_REPO}"/csi-provisioner:v2.2.2 "${K8S_IMAGE_REPO}"/csi-provisioner:v2.2.2
+    copy_image_to_cluster "${K8S_IMAGE_REPO}"/csi-node-driver-registrar:v2.2.0 "${K8S_IMAGE_REPO}"/csi-node-driver-registrar:v2.2.0
+    copy_image_to_cluster "${K8S_IMAGE_REPO}"/csi-resizer:v1.2.0 "${K8S_IMAGE_REPO}"/csi-resizer:v1.2.0
```

Comment:
This is really easy to forget to update. Please make it easier to see by setting the versions in a variable in `build.env` and use a current version when the variable is not set.

You will need to export the new variables in the `ci/centos` branch too

https://github.com/ceph/ceph-csi/blob/0786d159518286a87fe96d1bcc45f99871dd0ce6/single-node-k8s.sh#L58-L79